package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth0-deploy",
-  "version": "0.1.16",
+  "version": "0.1.17",
```

Comment:
can we leave this change out? the version is auto bumped by a release command.

Reply:
yea sure, I wasn't aware.